action.yml

```diff
@@ -0,0 +1,64 @@
+name: "Deep Security Smart Check"
+description: "Scan container images with Deep Security Smart Check."
+inputs:
+  DSSC_IMAGE_NAME:
+    description: "Container repository, eg myorg/myimage."
```

Comment:
again, `registryhost/myimage` might be more clear and consistent with the language used elsewhere in SmartCheck.

Reply:
Definitely agree!!

I think the ```registryhost/myimage``` is the best way to go...